.github/workflows/reusable_build_docker.yaml

```diff
@@ -0,0 +1,108 @@
+# This is a reusable workflow used by master and release CI
+on:
+  workflow_call:
+    inputs:
+      arch:
+        description: x86_64 or aarch64
+        required: true
+        type: string
+      bucket:
+        description: bucket suffix for packages
+        required: false
+        default: ''
+        type: string
+      version:
+        description: 'Falco version extracted from userspace/falco/config_falco.h'
+        required: true
+        type: string
+
+jobs:
+  build-docker:
```

Comment:
Here we just build all docker images as tarballs, then we upload all the tarballs to be later downloaded by `reusable_publish_docker` workflow.